CHANGELOG.md

```diff
@@ -3,10 +3,12 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 ### Added
-None.
+- Table method interpretation
+- Step editor
 
 ### Changed
 - translated all ui to english
+- Filefield upload instead of File-like entity type for GEF file.
```

Comment:
Changed upload from File-entity (with params from file) to using the FileField instead